test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.